<ui version="4.0" stdsetdef="1">
  <author>Peter Bartfai</author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>RotationCenterPanel</class>
  <widget class="QDialog" name="RotationCenterPanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>240</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>300</width>
        <height>240</height>
      </size>
    </property>
    <property name="baseSize">
      <size>
        <width>500</width>
        <height>240</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>LDView Rotation Center</string>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <property name="sizeGripEnabled">
      <bool>false</bool>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>4</number>
      </property>
      <item>
        <layout class="QGridLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item row="0" column="0">
            <widget class="QLabel" name="xLabel">
              <property name="text">
                <string>X:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="yLabel">
              <property name="text">
                <string>Y:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="zLabel">
              <property name="text">
                <string>Z:</string>
              </property>
              <property name="alignment">
                <set>Qt::AlignVCenter|Qt::AlignRight</set>
              </property>
              <property name="wordWrap">
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QLineEdit" name="xLine">
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QLineEdit" name="yLine">
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QLineEdit" name="zLine">
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
          <number>11</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item>
            <spacer name="SpacerReset1">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="resetButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Reset</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="SpacerReset2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
          <number>11</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item>
            <spacer name="SpacerButtons1">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>OK</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>5</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="SpacerButtons2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="4" margin="11"/>
  <pixmapfunction></pixmapfunction>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
